pom.xml

```diff
@@ -1810,6 +1810,12 @@
                 <artifactId>james-server-webadmin-data</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>${james.groupId}</groupId>
+                <artifactId>james-server-webadmin-integration-test-common</artifactId>
+                <version>${project.version}</version>
+                <type>test-jar</type>
```

Comment:
By having this as a test jar we loose dependency transitivity.

I would suggest moving `src/test` of this module to `src/main` so that we can only import the non-test-jar part of it, and save a lot of dependencies.